src/Core/Framework/Gateway/Context/Command/Handler/AddCustomerMessageCommandHandler.php

```diff
@@ -0,0 +1,26 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Core\Framework\Gateway\Context\Command\Handler;
+
+use Shopware\Core\Framework\Gateway\Context\Command\AbstractContextGatewayCommand;
+use Shopware\Core\Framework\Gateway\Context\Command\AddCustomerMessageCommand;
+use Shopware\Core\Framework\Gateway\GatewayException;
+use Shopware\Core\Framework\Log\Package;
+use Shopware\Core\System\SalesChannel\SalesChannelContext;
+
+#[Package('framework')]
+class AddCustomerMessageCommandHandler extends AbstractContextGatewayCommandHandler
+{
+    /**
+     * @param AddCustomerMessageCommand $command
+     */
+    public function handle(AbstractContextGatewayCommand $command, SalesChannelContext $context, array &$parameters): void
+    {
+        throw GatewayException::customerMessage($command->message);
+    }
+
+    public static function supportedCommands(): array
```

Comment:
I'm suprised PHPstan isn't complaining about this array not being documented

Reply:
@Bird87ZA becuse it is properly defined in the abstract parent